features/dd-sdk-android-trace/src/test/kotlin/com/datadog/android/trace/internal/domain/event/BigIntegerUtilsTest.kt

```diff
@@ -0,0 +1,198 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.trace.internal.domain.event
+
+import com.datadog.android.utils.forge.Configurator
+import fr.xgouchet.elmyr.annotation.StringForgery
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import java.math.BigInteger
+import java.util.concurrent.TimeUnit
+import kotlin.system.measureNanoTime
+
+@Extensions(
+    ExtendWith(ForgeExtension::class)
+)
+@ForgeConfiguration(Configurator::class)
+internal class BigIntegerUtilsTest {
+
+    private val testedUtils = BigIntegerUtils
+
+    @StringForgery(regex = "[a-f0-9]{32}")
+    lateinit var fakeTraceIdAsHexaString: String
+
+    @StringForgery(regex = "[a-f0-9]{16}")
+    lateinit var fakeHalfTraceIdHexaString: String
+
+    // region leastSignificantUnsignedLongAsHexa
+
+    @Test
+    fun `M use unsigned padded bits W extracting LSB from a max low trace id { hexa }`() {
```

Comment:
what `hexa` stands for? shouldn't we use just `hex`?

```suggestion
    fun `M use unsigned padded bits W extracting LSB from a max low trace id { hex }`() {
```

Reply:
yes this PR kind of got my headaches with namings :)